openghg/store/base/_base.py

```diff
@@ -243,12 +308,13 @@ def assign_data(
                 datasource.add_data(
                     metadata=meta_copy,
                     data=dataset,
+                    data_type=data_type,
+                    file_hashes=file_hashes,
```

Comment:
Just thinking about the places where `file_hashes` has had an empty dict passed (e.g. in _flux.py with the transform function) - would it be better to make this optional rather than required and then have to pass a dummy input?

Reply:
I've updated the other locations where an empty dictionary was being passed in. It was actually quite easy to get the data required so that worked out.